packages/manifold/contracts/gachaclaims/IERC1155GachaLazyClaim.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+import "./IGachaLazyClaim.sol";
+
+/**
+ * Gacha 1155 Lazy Claim interface
+ */
+interface IERC1155GachaLazyClaim is IGachaLazyClaim {
+  struct Claim {
+    StorageProtocol storageProtocol;
+    uint32 total;
+    uint32 totalMax;
+    uint48 startDate;
+    uint48 endDate;
+    uint112 startingTokenId;
```

Comment:
Make this a uint80 so it fits in one uint256.

storageProtocol = 8
total = 32
totalMax = 32
startDate = 48
endDate = 48
itemVariations = 8

total = 176
you have 80 left